client/jetpack-cloud/sections/partner-portal/license-bundle-card-description/index.tsx

```diff
@@ -10,83 +16,52 @@ interface Props {
 
 export default function LicenseBundleCardDescription( { product }: Props ) {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
 
-	let textDescription = '';
-	const features = [];
+	const { description, features } = useProductDescription( product.slug );
 
-	switch ( product.slug ) {
-		case 'jetpack-complete':
-			textDescription = translate( 'Includes all Security 1TB and full Jetpack package.' );
-			features.push(
-				translate( 'All Security products' ),
-				translate( '1TB cloud storage' ),
-				translate( 'Full Jetpack package' )
-			);
-			break;
-		case 'jetpack-security-t1':
-			textDescription = translate(
-				'Includes VaultPress Backup 10GB, Scan Daily and Akismet Anti-spam.'
-			);
-			features.push(
-				translate( 'VaultPress Backup 10GB' ),
-				translate( 'Scan Daily' ),
-				translate( 'Akismet Anti-spam*' )
-			);
-			break;
-		case 'jetpack-security-t2':
-			textDescription = translate(
-				'Includes VaultPress Backup 1TB, Scan Daily and Akismet Anti-spam.'
-			);
-			features.push(
-				translate( 'VaultPress Backup 1TB' ),
-				translate( 'Scan Daily' ),
-				translate( 'Akismet Anti-spam*' )
-			);
-			break;
-		case 'jetpack-anti-spam':
-			textDescription = translate( 'Automatically clear spam from your comments and forms.' );
-			break;
-		case 'jetpack-backup-t1':
-			textDescription = translate( 'Real-time cloud backups with one-click restores.' );
-			break;
-		case 'jetpack-backup-t2':
-			textDescription = translate( 'Real-time cloud backups with one-click restores.' );
-			break;
-		case 'jetpack-boost':
-			textDescription = translate(
-				'Essential tools to speed up your site - no developer required.'
-			);
-			break;
-		case 'jetpack-scan':
-			textDescription = translate( 'Automatic malware scanning with one-click fixes.' );
-			break;
-		case 'jetpack-videopress':
-			textDescription = translate(
-				'High-quality, ad-free video built specifically for WordPress.'
+	const [ showLightbox, setShowLightbox ] = useState( false );
+
+	const onShowLightbox = useCallback(
+		( e: any ) => {
```

Comment:
Nitpick: Can we replace this with `React.MouseEvent< HTMLElement >`?

Reply:
I agree, I should have done this. 👍 fix that one.